tools/xcodeproj-shims/swiftc-stub.sh

```diff
@@ -7,8 +7,9 @@ if [[ $# -eq 1 && $1 == "-v" ]]; then
     exec swiftc -v
 fi
 
+# Source of json-util: https://github.com/dominictarr/JSON.sh
 write_output_files() {
-    cat $1 | jq 'map(map(.)) | flatten | unique | .[]' -r | xargs touch
+    cat $1 | $(dirname "$0")/json-util -b -n  | sed 's/\[.*\]//' | xargs touch
```

Comment:
example of one line of output from json-util -b -n is:

[".../register/Frameworks/Chartography/DemoApp/ConfigurableViewController.swift","dependencies"] ".../DerivedData/Chartography-Project-dljjinukhucgryfdyxkxdhbgnsrk/Build/Intermediates.noindex/Chartography-Project.build/Debug-iphonesimulator/Chartography-DemoApp.build/Objects-normal/x86_64/ConfigurableViewController.d"

where -b denotes get to the leaf nodes and remove empty entries, -n means remove anything that has no "head"

With this output, the next step is to use sed 's/\[.*\]//' to remove the first portion of the output (the path to the value) and so we end up a list of values, which is what jq 'map(map(.)) | flatten | unique | .[]' -r is doing